trio/_threads.py

```diff
@@ -138,6 +138,7 @@ async def to_thread_run_sync(sync_fn, *args, cancellable=False, limiter=None):
 
     """
     await trio.lowlevel.checkpoint_if_cancelled()
+    cancellable = bool(cancellable)  # Sanitize for use in abort function
```

Comment:
I'm not sure the comment adds much, and I'd prefer if we could avoid ableist language here

Reply:
I do think a comment is needed to associate it with the abort function, otherwise it could look like an unnecessary step (even to my future self!) Maybe "normalize" or "canonicalize"?

Reply:
"Ensure is bool even if _bool_ not implemented"?

Reply:
Good suggestion, I went slightly broader in the amended and rebased commit

Reply:
Oh I understand what this does now, completely missed it with the previous comment